src/Utilities/Functional.hpp

```diff
@@ -217,7 +181,7 @@ struct GetArgument : Functional<Arity> {
 /// The identity higher order function object
 struct Identity : Functional<1> {
   template <class T>
-  constexpr const T& operator()(const T& t) noexcept {
+  inline constexpr const T& operator()(const T& t) noexcept {
```

Comment:
Again, inline doesn't do what you think it does. `inline` is a statement about (roughly) linkage, not about whether or not the function is inlined. If you actually want the compiler to inline this we have `SPECTRE_ALWAYS_INLINE` in `Utilities/ForceInline.hpp`.

[Here](https://en.cppreference.com/w/cpp/language/inline) is what inline actually does.